projects/vdk-control-cli/requirements.txt

```diff
@@ -6,7 +6,7 @@ pluggy~=0.13
 tabulate
 click-spinner
 
-vdk-control-service-api==1.0.5
+vdk-control-service-api==1.0.6
```

Comment:
Is the new API package released, as it does not seem to be in PYPI?

Reply:
That's why I added in the description "This change will be merged after #574.".